src/table/table.scss

```diff
@@ -20,7 +20,7 @@
   }
 
   @include iui-table-cell-icon;
-  @include iui-scroll-snapping;
+  @include iui-scroll-snapping($selector: '.iui-row');
```

Comment:
Does this not work?

```suggestion
  @include iui-scroll-snapping(.iui-row);
```

Reply:
This works: `@include iui-scroll-snapping('.iui-row');`. I'll make the change.